Mod/craft 5 compatible

anita-chouhan opened the pull request.

rungta reviewed (changes requested):
Please use the `craft-4` branch (v2 release) as the base and make sure we are retaining Craft 4 support as well.

rungta commented:
@anita-chouhan Please change the base branch to `craft-4` from `master`.

anita-chouhan commented:
> @anita-chouhan Please change the base branch to `craft-4` from `master`.

Earlier, I changed the base for the PR on the [glue-agency repository](https://github.com/glue-agency/craft-fingerprint-assets/pull/3). Sorry for the misunderstanding. I've added the upstream `craft-4` branch in this repository and changed the base now.

rungta commented on README.md:
```diff
@@ -1,9 +1,9 @@
 
-Craft CMS 4.0.0 or later.
\ No newline at end of file
+Craft CMS 4.0.0 or later, 5.6.0 or later.
```
Why only 5.6.0 and not 5.0.x onwards?

anita-chouhan replied:
I tested on the 5.6.0 version, so I added that. It can work on lower versions as well.

rungta replied:
In one place (composer.json) you’re saying the plugin only needs 5.0.0 or higher to work. In another place you’re saying the plugin needs 5.6.0 or higher to work. That is inconsistent. Better to test if this works on 5.0.0 as well.
